src/nwbuild.js

```diff
@@ -62,6 +63,10 @@ const nwbuild = async ({
     }
   }
 
+  if (cacheDir) {
+    await fs.mkdir(cacheDir, { recursive: true });
+  }
+
```

Comment:
Wow your solution is much more elegant than mine god damn

Reply:
Haha, seems straightforward enough.

I think my glob solution needs more work though. Currently needing to pass in the name of the file and it's contents glob pattern to exclude the file... 

Also have some more work in that renames the .app product once it's completed building, but need to test on windows

Reply:
> I think my glob solution needs more work though. Currently needing to pass in the name of the file and it's contents glob pattern to exclude the file...

I was thinking of a way to avoid a glob flag. For example: `./src` could imply `./src/**/*` if they don't have any specific globbing patterns in mind? To prevent a breaking change.we could pass multiple patterns (`./src ./assets/*`) as a string.

> Also have some more work in that renames the .app product once it's completed building, but need to test on windows

I'm actually working on that in #712